src/style.css

```diff
@@ -1,3 +1,95 @@
-body {
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap");
+
+:root {
+  --select-color: #00acdc;
+  --main-color: #000;
+  --font-family: roboto, sans-serif;
+  --main-text-size: 16px;
+}
+
+.body {
   margin: 0;
+  font-family: var(--font-family);
+  font-size: var(--main-text-size);
+  color: var(--main-color);
+  line-height: normal;
+}
+
+a {
+  text-decoration: none;
+  color: #000;
+}
+
+.nav-list__link {
+  align-items: center;
+  display: flex;
+  height: 100%;
+}
+
+.header__logo__image {
+  width: 40px;
+  height: 40px;
+}
+
+.nav-list {
+  display: flex;
+  flex-direction: row;
+  height: 100%;
+  align-items: center;
+  margin: 0;
+  padding: 0;
+}
+
+.nav-list__item {
+  list-style-type: none;
+  text-transform: uppercase;
+  font-size: 12px;
+  font-weight: 500;
+  align-items: center;
+  height: 100%;
+  display: flex;
+}
+
+.nav-list__link:hover {
+  color: var(--select-color);
+}
+
+.nav-list__item:not(:first-child) {
+  margin-left: 20px;
+}
+
+.nav-list__link.is-active {
+  position: relative;
+  color: var(--select-color);
+}
```

Comment:
A better idea is not to alter `position` based on the modifier

a better idea would be moving `position: relative` to `nav-list__link`